src/Enterspeed.Source.UmbracoCms/Enterspeed.Source.UmbracoCms.csproj

```diff
@@ -19,7 +19,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Enterspeed.Source.Sdk" Version="0.8.0" />
+		<PackageReference Include="Enterspeed.Source.Sdk" Version="1.0.0-alpha-1" />
```

Comment:
As we talked about, the version here should be changed back

Reply:
This has now been updated. :)